config/dm_config/kustomization.yaml

```diff
@@ -6,6 +6,6 @@ generatorOptions:
 
 configMapGenerator:
   - name: nnf-dm-config
-    namespace: default
+    namespace: nnf-dm-system
```

Comment:
You might want to add this folder to the `bases` array, then you _should_ get the name prefix "nnf-dm" applied to the config and the namespace auto-filled for you.

Then you could clean up the deploy.sh script.

You don't have to do this in this PR - but it would clean things up long-term.